includes/abstracts/abstract-wc-stripe-payment-gateway.php

```diff
@@ -561,6 +568,10 @@ public function process_response( $response, $order ) {
 			$this->update_fees( $order, is_string( $response->balance_transaction ) ? $response->balance_transaction : $response->balance_transaction->id );
 		}
 
+		if ( isset( $response->payment_method_details ) && isset( $response->payment_method_details->card ) && isset( $response->payment_method_details->card->mandate ) ) {
```

Comment:
Maybe we can simplify this a little:

```suggestion
		if ( isset( $response->payment_method_details->card->mandate ) ) {
```

Reply:
Nice, I didn't know this was possible! 🎉

Reply:
Applied in d13134b491a3a9e0eaaecd59e190f17879767346